sympy/utilities/lambdify.py

```diff
@@ -88,22 +90,74 @@
     "Matrix": "array",
     "MutableDenseMatrix": "array",
     "ImmutableMatrix": "array",
+    "ImmutableDenseMatrix": "array"
+
+}
+
+# most of the functions of scipy is similar to numpy. Checked each functions.
+SCIPY_TRANSLATIONS = {
+    "acos": "arccos",
+    "acosh": "arccosh",
+    "arg": "angle",
+    "asin": "arcsin",
+    "asinh": "arcsinh",
+    "atan": "arctan",
+    "atan2": "arctan2",
+    "atanh": "arctanh",
+    "ceiling": "ceil",
+    "E": "e",
+    "im": "imag",
+    "ln": "log",
+    "Max": "amax",
+    "Min": "amin",
+    "Mod": "mod",
+    "oo": "inf",
+    "re": "real",
+    "SparseMatrix": "array",
+    "ImmutableSparseMatrix": "array",
+    "Matrix": "array",
+    "MutableDenseMatrix": "array",
+    "ImmutableMatrix": "array",
     "ImmutableDenseMatrix": "array",
+    "elliptic_k": "ellipk",
+    "elliptic_f": "ellipeinc",
+    "besselj": "jn",
+    "bessely": "yn",
+    "besseli": "iv",
+    "besselk": "kn",
+    "loggamma": "gammaln",
+    "digamma": "psi",
+    "hyper": "hyp1f1",
 }
 
-NUMEXPR_TRANSLATIONS = {}
+NUMEXPR_TRANSLATIONS = {
+# Found need of mapping for numexpr,since supported functions: https://github.com/pydata/numexpr#supported-functions 
+    "Abs": "abs",
+    "acos": "arccos",
+    "acosh": "arccosh",
+    "arg": "angle",
+    "asin": "arcsin",
+    "asinh": "arcsinh",
+    "atan": "arctan",
+    "atan2": "arctan2",
+    "atanh": "arctanh",
+    "ln": "log",
+    "re": "real",
+    "im": "imag",
+}
 
 # Available modules:
 MODULES = {
     "math": (MATH, MATH_DEFAULT, MATH_TRANSLATIONS, ("from math import *",)),
     "mpmath": (MPMATH, MPMATH_DEFAULT, MPMATH_TRANSLATIONS, ("from mpmath import *",)),
     "numpy": (NUMPY, NUMPY_DEFAULT, NUMPY_TRANSLATIONS, ("import_module('numpy')",)),
+    # according to the external/importtools.py we should use import_module() for numpy,scipy
+    "scipy": (SCIPY, SCIPY_DEFAULT, SCIPY_TRANSLATIONS, ("import_module('scipy')",)),
```

Comment:
Import_module import and return a module if it is installed.So I think it imports all the functions just like numpy ,otherwise we need to add `from scipy import *`